cherab/openadas/repository/atomic.py

```diff
@@ -108,6 +108,47 @@ def update_recombination_rates(rates, repository_path=None):
         _update_and_write_adf11(species, rate_data, path)
 
 
+def add_thermalchargeexchange_rate(species, charge, rate, repository_path=None):
+    """
+    Adds a single thermal charge exchange rate to the repository.
+
+    If adding multiple rates, consider using the update_recombination_rates()
+    function instead. The update function avoids repeatedly opening and closing
+    the rate files.
+
+    :param repository_path:
+    :return:
+    """
+
+    update_thermalchargeexchange_rates({
+        species: {
+            charge: rate
+        }
+    }, repository_path)
+
+
+def update_thermalchargeexchange_rates(rates, repository_path=None):
+    """
+    Thermal charge exchange rate file structure
+
+    /recombination/<species>.json
```

Comment:
Follow the path conventions of the other CX rate, the beam_cx_pec rate. Depends on if you end up needing donor_charge, you might not need it.

cx/donor_ion/donor_charge/receiver_ion/receiver_charge.json